jfrog-applications/jfrog-cli/cli-for-jfrog-artifactory/generic-files.md

```diff
@@ -264,6 +264,13 @@ Copy all artifacts located under **/rabbit** in the **source-frog-repo** reposit
 jf rt cp "source-frog-repo/rabbit/*" target-frog-repo/rabbit/ --props=Version=1.0
 ```
 
+#### Example 4
+
+Copy all artifacts located under /rabbit in the **source-frog-repo** repository to an absulote path.
```

Comment:
Since it's a bit hard to understand from the example description what the desired outcome is, I suggest adding a "before" and "after" example that demonstrates the outcome. Also, I'm not sure "to an absolute path" isn't fully clear, so I suggest rephrasing the entire description.